### DISCUSSION TITLE
Kubernetes Cluster Downtime Issue After Resetting Old Master Node Following Replacement with New Master Node

### DISCUSSION THREAD

**Discussion by Bangar89** [kubernetes/website#47232] [General]
successfully joined the new master node to the existing Kubernetes cluster and managed to drain and delete the old master node. However, when using the kubeadm reset command to clean the old master node, the cluster goes down.
 
To join the new master node to the existing cluster, I used the token and certificate key from the old master node.
 
While installing the control plane components on the new master node, I specified the Load Balancer and initial node IP as the new master node name and updated the Flannel configuration accordingly. Despite these steps, the cluster still goes down after resetting the old master node.

**Comment by stmcginnis** [comment]
This doesn't sound like a website issue. Is this related to a tutorial you were following on the site?

**Comment by sftim** [comment]
GitHub is not the right place for support requests.

If you're looking for help, check [Server Fault](https://serverfault.com/questions/tagged/kubernetes).

You can also post your question on the [Kubernetes Slack](http://slack.k8s.io/) or the [Discuss Kubernetes](https://discuss.kubernetes.io/) forum.
